tonic_validate/utils/telemetry.py

```diff
@@ -79,6 +116,9 @@ def log_benchmark(self, num_of_questions: int):
         )
 
     def link_user(self):
+        """
+        Links an api user to sdk user in telemetry to be able to filter out users not on the web UI.
```

Comment:
In the telemetry, links an API user to an SDK user. Used to filter out users that do not use the web application.